### ISSUE TITLE
How to pre-training?

### ISSUE THREAD

**Issue by woans0104** [AlibabaResearch/AdvancedLiterateMachinery#40]
Hi. Thanks for the great research.

I would like to train GeoLayoutLM using a dataset consisting of other languages, is it possible to provide a pre-trainable code? Or could you give me a tip, for pre-training, I think I need the loss configurations for these 3: GeoPair, GeoMPair and GeoTriplet
